skypy/utils/random.py

```diff
@@ -11,11 +11,12 @@
    :toctree: ../api/
 
    schechter
-
```

Comment:
The name of the additional functions should appear here, right after `schechter`  

```
   .. autosummary::
   :nosignatures:
    :toctree: ../api/

    schechter
    conditional_prob_shmr
```
 However, I think there are plans to change this structure. Hopefully, the infrastructure team can tell us.